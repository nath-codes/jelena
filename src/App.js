import React, { Component } from "react";
import { ThemeProvider, injectGlobal } from "styled-components";
import theme from "./constants/theme";
import Hero from "./components/Hero";

injectGlobal`
  html {
    box-sizing: border-box;
  }

  *,
  *::before,
  *::after {
    box-sizing: inherit;
  }

  body {
    color: '#333';
    font-family: 'montserrat';
    font-size: 16px;
    line-height: 1.5;
  }

  a {
    text-decoration: none;
  }

  img {
    height: auto;
    max-width: 100%;
  }
`;

class App extends Component {
  render() {
    return (
      <ThemeProvider theme={theme}>
        <div className="App">
          <Hero />
        </div>
      </ThemeProvider>
    );
  }
}

export default App;
