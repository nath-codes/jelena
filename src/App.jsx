import React, { Component } from "react";
import { ThemeProvider } from "styled-components";
import theme from "./constants/theme";
import Frame from "./components/Frame";
import Hero from "./components/Hero";
import Rsvp from "./components/Rsvp";
import Timetable from "./components/Timetable";
import Location from "./components/Location";
import Gifts from "./components/Gifts";
import Header from "./components/Header";
import "./constants/inject-global";

class App extends Component {
  render() {
    return (
      <ThemeProvider theme={theme}>
        <div className="App">
          <Header />
          <Frame position="top-left" />
          <Frame position="top-right" />
          <Frame position="bottom-left" />
          <Frame position="bottom-right" />
          <Hero />
          <Rsvp />
          <Location />
          <Timetable />
          <Gifts />
        </div>
      </ThemeProvider>
    );
  }
}

export default App;
