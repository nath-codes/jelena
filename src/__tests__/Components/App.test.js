import React from "react";
import renderer from "react-test-renderer";
import "jest-styled-components";
import App from "../../App";

const matchMediaMock = () => {
  return {
    matches: false,
    addListener: function() {},
    removeListener: function() {}
  };
};

window.matchMedia = window.matchMedia || matchMediaMock;

describe("<App />", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<App />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
