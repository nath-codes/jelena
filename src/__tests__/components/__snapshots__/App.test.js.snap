// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> renders correctly 1`] = `
.c2 {
  background-image: url(tile-xs.png);
  background-repeat: no-repeat;
  background-size: contain;
  height: 125px;
  position: fixed;
  width: 125px;
  left: 0;
  top: 0;
}

.c3 {
  background-image: url(tile-xs.png);
  background-repeat: no-repeat;
  background-size: contain;
  height: 125px;
  position: fixed;
  width: 125px;
  right: 0;
  top: 0;
  -webkit-transform: rotate(90deg);
  -ms-transform: rotate(90deg);
  transform: rotate(90deg);
}

.c4 {
  background-image: url(tile-xs.png);
  background-repeat: no-repeat;
  background-size: contain;
  height: 125px;
  position: fixed;
  width: 125px;
  bottom: 0;
  left: 0;
  -webkit-transform: rotate(270deg);
  -ms-transform: rotate(270deg);
  transform: rotate(270deg);
}

.c5 {
  background-image: url(tile-xs.png);
  background-repeat: no-repeat;
  background-size: contain;
  height: 125px;
  position: fixed;
  width: 125px;
  bottom: 0;
  right: 0;
  -webkit-transform: rotate(180deg);
  -ms-transform: rotate(180deg);
  transform: rotate(180deg);
}

.c11 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  min-height: 100vh;
  margin: 0 auto;
  padding: 2.4rem 0;
  position: relative;
  width: 80%;
}

.c8 {
  color: #1b56c5;
  font-family: bebas-neue;
  font-size: 2.4rem;
  font-weight: 400;
  line-height: 1;
  margin-bottom: 1.2rem;
  text-align: center;
}

.c8:last-child {
  margin-bottom: 0;
}

.c9 {
  color: #1b56c5;
  font-family: bebas-neue;
  font-size: 3.5999999999999996rem;
  font-weight: 400;
  line-height: 1;
  margin-bottom: 1.2rem;
  text-align: center;
}

.c9:last-child {
  margin-bottom: 0;
}

.c7 {
  border: 0;
  -webkit-clip: rect(0 0 0 0);
  clip: rect(0 0 0 0);
  -webkit-clip-path: inset(50%);
  clip-path: inset(50%);
  height: 1px;
  margin: -1px;
  overflow: hidden;
  padding: 0;
  position: absolute;
  white-space: nowrap;
  width: 1px;
}

.c6 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  min-height: 100vh;
  margin: 0 auto;
  padding: 2.4rem 0;
  position: relative;
  width: 80%;
}

.c10 {
  height: 4.8rem;
  margin-bottom: 1.2rem;
}

.c12 {
  color: #1b56c5;
  font-family: bebas-neue;
  font-size: 3.5999999999999996rem;
  font-weight: 400;
  line-height: 3.5999999999999996rem;
  margin: 0 0 2.4rem;
  text-align: center;
}

.c19 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  font-family: montserrat;
  font-size: 0.8999999999999999rem;
  width: 100%;
}

.c19 > span {
  font-size: 12px;
  margin-left: auto;
  padding-left: 6px;
}

.c15 {
  border: solid 3px #1b56c5;
  border-radius: 3px;
  font-family: montserrat;
  font-size: 0.8999999999999999rem;
  height: 2.4rem;
  padding: 0 0 0 0.6rem;
  width: 100%;
}

.c20 {
  margin-right: 0.6rem;
}

.c14 {
  border: none;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  height: auto;
  margin: 0 0 2.4rem;
  padding: 0;
}

.c14 [object Object]::after {
  clear: both;
  content: "";
  display: table;
}

.c16 {
  color: #1b56c5;
  font-family: bebas-neue;
  font-size: 1.7999999999999998rem;
  line-height: 2.4rem;
  margin: 0 0 1.2rem;
}

.c22 {
  -webkit-align-self: center;
  -ms-flex-item-align: center;
  align-self: center;
  background: #1b56c5;
  border: 0;
  color: #fff;
  cursor: pointer;
  font-family: bebas-neue;
  font-size: 1.2rem;
  line-height: 1;
  padding-bottom: 0.6rem;
  padding-top: 0.6rem;
  padding-left: 2.4rem;
  padding-right: 2.4rem;
  position: relative;
  -webkit-transition: all 0.2s;
  transition: all 0.2s;
}

.c17 {
  list-style-type: none;
  padding: 0;
  margin: 0;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  width: 100%;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
}

.c17 li:nth-child(2) {
  margin-left: 1.2rem;
}

.c18 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  height: 1.2rem;
  margin-bottom: 0.6rem;
  margin-bottom: 0;
}

.c18:last-child {
  margin-bottom: 0;
}

.c21 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  height: 1.2rem;
  margin-bottom: 0.6rem;
}

.c21:last-child {
  margin-bottom: 0;
}

.c24 {
  -webkit-align-self: center;
  -ms-flex-item-align: center;
  align-self: center;
  font-size: 0.8999999999999999rem;
  line-height: 1.3;
  max-width: 33.6rem;
}

.c13 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  max-width: 28.799999999999997rem;
  width: 100%;
}

.c13 .message {
  margin: 1.2rem 0;
}

.c28 {
  list-style-type: none;
  padding: 0;
  margin: 0;
}

.c27 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  min-height: 100vh;
  margin: 0 auto;
  padding: 2.4rem 0;
  position: relative;
  width: 80%;
  padding-bottom: 0;
}

.c29 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  font-size: 0.8999999999999999rem;
  margin-bottom: 2.4rem;
  text-align: center;
}

.c29:last-child {
  margin-bottom: 0;
}

.c29 .time {
  color: #1b56c5;
  margin-bottom: 1.2rem;
  min-width: 5.3999999999999995rem;
  white-space: nowrap;
}

.c23 {
  border: 0;
  -webkit-filter: grayscale(1);
  filter: grayscale(1);
  height: 14.399999999999999rem;
  margin-bottom: 2.4rem;
  width: 100%;
}

.c25 {
  margin: 0 0 1.2rem;
}

.c25:last-child {
  margin-bottom: 0;
}

.c26 {
  color: #1b56c5;
  position: relative;
  -webkit-text-decoration: none;
  text-decoration: none;
}

.c26::before {
  background-color: #fff;
  bottom: -3px;
  content: "";
  height: 2px;
  left: 0;
  position: absolute;
  -webkit-transform: scaleX(0);
  -ms-transform: scaleX(0);
  transform: scaleX(0);
  -webkit-transition: -webkit-transform 0.3s cubic-bezier(0.4,0,0.2,1);
  -webkit-transition: transform 0.3s cubic-bezier(0.4,0,0.2,1);
  transition: transform 0.3s cubic-bezier(0.4,0,0.2,1);
  width: 100%;
}

.c26:hover::before {
  background-color: #1b56c5;
  -webkit-transform: scaleX(1);
  -ms-transform: scaleX(1);
  transform: scaleX(1);
}

.c31 {
  -webkit-align-self: center;
  -ms-flex-item-align: center;
  align-self: center;
  background: #1b56c5;
  border: 0;
  color: #fff;
  cursor: pointer;
  font-family: bebas-neue;
  font-size: 1.2rem;
  line-height: 1;
  padding-bottom: 0.6rem;
  padding-top: 0.6rem;
  padding-left: 2.4rem;
  padding-right: 2.4rem;
  position: relative;
  -webkit-transition: all 0.2s;
  transition: all 0.2s;
}

.c30 {
  -webkit-align-self: center;
  -ms-flex-item-align: center;
  align-self: center;
  font-size: 0.8999999999999999rem;
  line-height: 1.3;
  max-width: 33.6rem;
  margin-bottom: 2.4rem;
}

.c1 {
  color: #1b56c5;
  position: relative;
  -webkit-text-decoration: none;
  text-decoration: none;
  font-family: bebas-neue;
  font-size: 1.2rem;
  margin-right: 1.2rem;
}

.c1::before {
  background-color: #fff;
  bottom: -3px;
  content: "";
  height: 2px;
  left: 0;
  position: absolute;
  -webkit-transform: scaleX(0);
  -ms-transform: scaleX(0);
  transform: scaleX(0);
  -webkit-transition: -webkit-transform 0.3s cubic-bezier(0.4,0,0.2,1);
  -webkit-transition: transform 0.3s cubic-bezier(0.4,0,0.2,1);
  transition: transform 0.3s cubic-bezier(0.4,0,0.2,1);
  width: 100%;
}

.c1:hover::before {
  background-color: #1b56c5;
  -webkit-transform: scaleX(1);
  -ms-transform: scaleX(1);
  transform: scaleX(1);
}

.c1:last-child {
  margin-right: 0;
}

.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  margin: 2.4rem 0 0;
}

@media (-webkit-min-device-pixel-ratio:2) {
  .c2 {
    background-image: url(tile-xs@2x.png);
  }
}

@media (min-width:576px) {
  .c2 {
    background-image: url(tile-s.png);
    height: 150px;
    width: 150px;
  }
}

@media (min-width:576px) and (-webkit-min-device-pixel-ratio:2) {
  .c2 {
    background-image: url(tile-s@2x.png);
  }
}

@media (min-width:768px) {
  .c2 {
    background-image: url(tile-m.png);
    height: 200px;
    width: 200px;
  }
}

@media (min-width:768px) and (-webkit-min-device-pixel-ratio:2) {
  .c2 {
    background-image: url(tile-m@2x.png);
  }
}

@media (min-width:992px) {
  .c2 {
    background-image: url(tile-l.png);
    height: 250px;
    width: 250px;
  }
}

@media (min-width:992px) and (-webkit-min-device-pixel-ratio:2) {
  .c2 {
    background-image: url(tile-l@2x.png);
  }
}

@media (-webkit-min-device-pixel-ratio:2) {
  .c3 {
    background-image: url(tile-xs@2x.png);
  }
}

@media (min-width:576px) {
  .c3 {
    background-image: url(tile-s.png);
    height: 150px;
    width: 150px;
  }
}

@media (min-width:576px) and (-webkit-min-device-pixel-ratio:2) {
  .c3 {
    background-image: url(tile-s@2x.png);
  }
}

@media (min-width:768px) {
  .c3 {
    background-image: url(tile-m.png);
    height: 200px;
    width: 200px;
  }
}

@media (min-width:768px) and (-webkit-min-device-pixel-ratio:2) {
  .c3 {
    background-image: url(tile-m@2x.png);
  }
}

@media (min-width:992px) {
  .c3 {
    background-image: url(tile-l.png);
    height: 250px;
    width: 250px;
  }
}

@media (min-width:992px) and (-webkit-min-device-pixel-ratio:2) {
  .c3 {
    background-image: url(tile-l@2x.png);
  }
}

@media (-webkit-min-device-pixel-ratio:2) {
  .c4 {
    background-image: url(tile-xs@2x.png);
  }
}

@media (min-width:576px) {
  .c4 {
    background-image: url(tile-s.png);
    height: 150px;
    width: 150px;
  }
}

@media (min-width:576px) and (-webkit-min-device-pixel-ratio:2) {
  .c4 {
    background-image: url(tile-s@2x.png);
  }
}

@media (min-width:768px) {
  .c4 {
    background-image: url(tile-m.png);
    height: 200px;
    width: 200px;
  }
}

@media (min-width:768px) and (-webkit-min-device-pixel-ratio:2) {
  .c4 {
    background-image: url(tile-m@2x.png);
  }
}

@media (min-width:992px) {
  .c4 {
    background-image: url(tile-l.png);
    height: 250px;
    width: 250px;
  }
}

@media (min-width:992px) and (-webkit-min-device-pixel-ratio:2) {
  .c4 {
    background-image: url(tile-l@2x.png);
  }
}

@media (-webkit-min-device-pixel-ratio:2) {
  .c5 {
    background-image: url(tile-xs@2x.png);
  }
}

@media (min-width:576px) {
  .c5 {
    background-image: url(tile-s.png);
    height: 150px;
    width: 150px;
  }
}

@media (min-width:576px) and (-webkit-min-device-pixel-ratio:2) {
  .c5 {
    background-image: url(tile-s@2x.png);
  }
}

@media (min-width:768px) {
  .c5 {
    background-image: url(tile-m.png);
    height: 200px;
    width: 200px;
  }
}

@media (min-width:768px) and (-webkit-min-device-pixel-ratio:2) {
  .c5 {
    background-image: url(tile-m@2x.png);
  }
}

@media (min-width:992px) {
  .c5 {
    background-image: url(tile-l.png);
    height: 250px;
    width: 250px;
  }
}

@media (min-width:992px) and (-webkit-min-device-pixel-ratio:2) {
  .c5 {
    background-image: url(tile-l@2x.png);
  }
}

@media (min-width:576px) {
  .c19 > span {
    margin-left: 0;
  }
}

@media (min-width:768px) {
  .c19 > span {
    font-size: 14px;
  }
}

@media (min-width:768px) {
  .c24 {
    line-height: 1.4;
  }
}

@media (min-width:992px) {
  .c24 {
    line-height: 1.5;
  }
}

@media (min-width:768px) {
  .c28 {
    margin-left: 7.8rem;
  }
}

@media (min-width:768px) {
  .c29 {
    -webkit-flex-direction: row;
    -ms-flex-direction: row;
    flex-direction: row;
    margin-bottom: 1.7999999999999998rem;
    text-align: left;
  }

  .c29 .time {
    margin-bottom: 0;
    white-space: nowrap;
  }
}

@media (min-width:768px) {
  .c30 {
    line-height: 1.4;
  }
}

@media (min-width:992px) {
  .c30 {
    line-height: 1.5;
  }
}

@media (min-width:576px) {
  .c0 {
    margin-top: 1.2rem 0 0;
  }
}

<div
  className="App"
>
  <header
    className="c0"
  >
    <nav
      className=""
    >
      <a
        className="c1"
        href="#rsvp"
      >
        RSVP
      </a>
      <a
        className="c1"
        href="#location"
      >
        Location
      </a>
      <a
        className="c1"
        href="#timetable"
      >
        Timetable
      </a>
      <a
        className="c1"
        href="#gifts"
      >
        Gifts
      </a>
    </nav>
  </header>
  <div
    className="c2"
  />
  <div
    className="c3"
  />
  <div
    className="c4"
  />
  <div
    className="c5"
  />
  <section
    className="c6"
  >
    <h1
      className="c7"
    >
      Jelena wedding information
    </h1>
    <div
      className="c8"
    >
      The wedding of
    </div>
    <div
      className="c9"
      size="large"
    >
      Joseph David
      <br />
      Goldsworthy
    </div>
    <div
      className="c8"
    >
      &
    </div>
    <div
      className="c9"
      size="large"
    >
      Helena Maria
      <br />
      Malkowska Zaba
    </div>
    <div
      className="c10"
    >
      <img
        alt=""
        src="rings.png"
      />
    </div>
  </section>
  <section
    className="c11"
    id="rsvp"
  >
    <h2
      className="c12"
    >
      RSVP
    </h2>
    <form
      className="c13"
      onSubmit={[Function]}
    >
      <div
        className="c14"
      >
        <input
          className="c15"
          name="fullName"
          onChange={[Function]}
          placeholder="Enter your name here"
          required="required"
          type="text"
          value=""
        />
      </div>
      <h3
        className="c16"
      >
        Attending
      </h3>
      <div
        className="c14"
      >
        <ul
          className="c17"
        >
          <li
            className="c18"
          >
            <label
              className="c19"
            >
              <input
                className="c20"
                name="attending"
                onChange={[Function]}
                type="radio"
                value="Yes"
              />
              Yes
            </label>
          </li>
          <li
            className="c21"
          >
            <label
              className="c19"
            >
              <input
                className="c20"
                name="attending"
                onChange={[Function]}
                type="radio"
                value="No"
              />
              No
            </label>
          </li>
        </ul>
      </div>
      <button
        className="c22"
        type="submit"
      >
        Submit
      </button>
    </form>
  </section>
  <section
    className="c11"
    id="location"
  >
    <h2
      className="c12"
    >
      Location
    </h2>
    <iframe
      className="c23"
      src="https://www.google.com/maps/embed/v1/place?key=AIzaSyA0eUK8JOPmTAbb7mmk-Q59ioYz6VfvKlg&q=place_id:ChIJW9LhQoWScUgRpUZbE5MC6tY"
    />
    <div
      className="c24"
    >
      <p
        className="c25"
      >
        <a
          className="c26"
          href="http://kingswestonhouse.co.uk/"
        >
          Kings Weston House
        </a>
         is approximately fifteen minutes' drive from central Bristol, Temple Meads train station and near to Blaise Castle Estate. There is a free car park in the house grounds. If you need a taxi from the centre, we would recommend calling V Cars on
         
        <a
          className="c26"
          href="tel:0117 9252626"
        >
          0117 9252626
        </a>
         or using Uber.
      </p>
      <p
        className="c25"
      >
        If you need to spend the night in Bristol before or after the wedding then please ask us for our recommended places to stay.
      </p>
    </div>
  </section>
  <section
    className="c27"
    id="timetable"
  >
    <h2
      className="c12"
    >
      Timetable
    </h2>
    <ul
      className="c28"
    >
      <li
        className="c29"
      >
        <div
          className="time"
        >
          1:30 pm
        </div>
        <div
          className="title"
        >
          Arrive
        </div>
      </li>
      <li
        className="c29"
      >
        <div
          className="time"
        >
          2:00 pm
        </div>
        <div
          className="title"
        >
          Ceremony
        </div>
      </li>
      <li
        className="c29"
      >
        <div
          className="time"
        >
          2:30 pm
        </div>
        <div
          className="title"
        >
          Reception drinks
        </div>
      </li>
      <li
        className="c29"
      >
        <div
          className="time"
        >
          3:45 pm
        </div>
        <div
          className="title"
        >
          Wedding breakfast
        </div>
      </li>
      <li
        className="c29"
      >
        <div
          className="time"
        >
          5:45 pm
        </div>
        <div
          className="title"
        >
          Speeches
        </div>
      </li>
      <li
        className="c29"
      >
        <div
          className="time"
        >
          7:00 pm
        </div>
        <div
          className="title"
        >
          First dance
        </div>
      </li>
      <li
        className="c29"
      >
        <div
          className="time"
        >
          8:30 pm
        </div>
        <div
          className="title"
        >
          Cutting of the (cheese) cake
        </div>
      </li>
      <li
        className="c29"
      >
        <div
          className="time"
        >
          Midnight
        </div>
        <div
          className="title"
        >
          Carriages
        </div>
      </li>
    </ul>
  </section>
  <section
    className="c11"
    id="gifts"
  >
    <h2
      className="c12"
    >
      Gifts
    </h2>
    <div
      className="c30"
    >
      <p
        className="c25"
      >
        Simply by being at our wedding, you will be getting us a wonderful gift – most of all, we want to spend the day with everyone we love and for you to celebrate with us.
      </p>
      <p
        className="c25"
      >
        Should you want to give us anything extra, however, we will be going to the beautiful archipelago of the Azores on our honeymoon and would really appreciate any contribution to help us on our adventures.
      </p>
      <p
        className="c25"
      >
        If you would like to make a gift to our honeymoon fund you can do so via PayPal using the button below or by calling our travel agent, Azores Choice, on 
        <a
          className="c26"
          href="tel:01768 721020"
        >
          01768 721020
        </a>
      </p>
    </div>
    <a
      className="c31"
      href="https://www.paypal.com/pools/c/858WNidwXt"
    >
      Contribute
    </a>
  </section>
</div>
`;
